src/main/groovy/com/morpheusdata/commvault/CommvaultPlugin.groovy

```diff
@@ -70,4 +73,33 @@ class CommvaultPlugin extends Plugin {
         }
         return backupProvider
     }
+
+    def captureActiveSubclientBackup(authConfig, subclientId, clientId, backupsetId) {
+        def activeJobsResults = CommvaultBackupUtility.getBackupJobs(authConfig, subclientId, [query: [clientId: clientId, backupsetId: backupsetId, jobFilter: "Backup", jobCategory: "Active"]])
+        if(activeJobsResults.success && activeJobsResults.results.size() > 0) {
+            def activeBackupJob = activeJobsResults.results.find { it.clientId == clientId && it.subclientId == subclientId && it.backupsetId == backupsetId }
+            return ServiceResponse.success([backupJobId: activeBackupJob.backupJobId])
+        }
+    }
+
+    def getBackupStatus(backupState) {
```

Comment:
Move this method to the `CommvaultComputeUtility` and rename the `CommvaultComputeUtility` to `CommvaultReferenceUtility`.

Reply:
@ddeyoung changes has been implemented as suggested.